cmd/root.go

```diff
@@ -131,11 +131,19 @@ func displayDB(rows *sql.Rows) error {
 		if err != nil {
 			return err
 		}
-	//TODO: switch between table and csv dependent on num columns(suggested num for table 5<=
+
+	// Switch between table and csv dependent on num columns(suggested num for table 5<=
 	default:
-		err := tableDisplay(rows)
-		if err != nil {
-			return err
+		if len(rows.Columns()) <= 5 {
```

Comment:
try 
```
columns, err := rows.Columns 
if err !=nil { return err}
if len(columns) <=5{....
```
or use a func returning a boolean for the if. That should clear up the vet & build problems

Reply:
Thanks, doing that now